### ISSUE TITLE
React sample project live updating isn't working

### ISSUE THREAD

**Issue by jsayres** [fsprojects/Fable#122]
I tried out the react sample project, and while it built fine, when changing the components.fs file in debug mode, the page in the browser didn't update.

Looking more deeply, it seems that in watch mode, the compiler isn't outputting an updated components.js file, although the sourcemap does change, strangely. When watching is stopped and the fable does a simple one-shot compile, the output is ok.

I'm on Mac El Capitan, mono version 4.2.3.

**Comment by alfonsogarciacaro** [comment]
Strange, it's working for me (I'm also on Mac, mono version is 4.4, not sure if it makes a difference). Could you please provide the following information to pinpoint the problem?

- Fable version (you can use `fable --help` to check)
- The exact command used to trigger the watch compilation
- Are you using the last version of the React sample from the repo?
- Which part of the code are you editing? (seems some parts cannot be hot-reloaded)

You can also try opening the Developer Tools in the browser. When hot reloading works fine you should see something like the following in the console:

```
[WDS] Hot Module Replacement enabled.
[WDS] App updated. Recompiling...
[WDS] App hot update...
[HMR] Checking for updates on the server...
[HMR] Updated modules:
[HMR]  - 244
[HMR] App is up to date.
```

Thanks a lot in advance for your help!

**Comment by jsayres** [comment]
The issue was that the HMR was saying no changes, and when I looked at the out folder, I saw that the components.js file wasn't being updated when I changed the F# source. I had the issue with samples zip download from the front page.

However, I just tried cloning the repo and using the samples folder in there, and it worked fine. Then I went back to the older one, and that now works as well.

I'm using vim, and I suspect it may be a temporary file issue. I've had issues with vim and file watchers on windows, but not on mac so far.

I think this issue can be closed for now unless I can reproduce the issue and better isolate the cause. Keep up the great work on this project!
